Docker/heasoft-caldb-wcstools.dockerfile

```diff
@@ -35,8 +34,8 @@ RUN \
     && rm goodfiles_swift_uvota.tar.Z \
     && caldbinfo INST swift uvota \
     && /bin/echo 'export CALDB='$CALDB >> /home/heasoft/.profile \
-    && /bin/echo "source $CALDB/software/tools/caldbinit.sh" >> /home/heasoft/.profile \
+    && /bin/echo "source $CALDB/caldbinit.sh" >> /home/heasoft/.profile \
     && /bin/echo 'export CALDB='$CALDB >> /home/heasoft/.bashrc \
-    && /bin/echo "source $CALDB/software/tools/caldbinit.sh" >> /home/heasoft/.bashrc \
+    && /bin/echo "source $CALDB/caldbinit.sh" >> /home/heasoft/.bashrc \
     && /bin/echo 'setenv CALDB '$CALDB >> /home/heasoft/.cshrc \
-    && /bin/echo "source $CALDB/software/tools/caldbinit.csh" >> /home/heasoft/.cshrc
+    && /bin/echo "source $CALDB/caldbinit.csh" >> /home/heasoft/.cshrc
```

Comment:
caldb seems to have removed the software/tools subdirectories and now the caldbinit script is just located in the base caldb folder